operatorcourier/validate.py

```diff
@@ -167,20 +172,54 @@ def _csv_spec_validation(self, spec, bundleData):
                     pass
 
             if "owned" not in customresourcedefinitions:
-                self._log_error("spec.customresourcedefinitions.owned"
+                self._log_error("spec.customresourcedefinitions.owned "
                                 "not defined for csv")
-                valid = False
-            else:
-                for crd in customresourcedefinitions["owned"]:
-                    if "name" not in crd:
-                        self._log_error("name not defined for item in "
-                                        "spec.customresourcedefinitions.")
-                        valid = False
-                    else:
-                        if crd["name"] not in crdList:
-                            self._log_error("custom resource definition referenced "
-                                            "in csv not defined in root list of crds")
+                return False
+
+            for csvOwnedCrd in customresourcedefinitions["owned"]:
```

Comment:
nit: Are we sticking to a convention on variable and function naming? If so, what is it? Snake case for functions and CamelCase for variables, etc?

Reply:
I think this is something we need to fix in the future. Because I have seen both styles in our code, so for now I just pick the style that the code nearby is using.

Reply:
Fair enough